yolo_detector:
  ros__parameters:
    model_path: "yolov8n.pt"
    target_class: "person"
    conf_thres: 0.5
    iou_thres: 0.45
    image_topic: "/camera/image_raw"
    bbox_topic: "/yolo_follower/bboxes"
    debug_image_topic: "/yolo_follower/debug_image"
    enable_debug_image: true

yolo_follower:
  ros__parameters:
    bbox_topic: "/yolo_follower/bboxes"
    cmd_vel_topic: "/cmd_vel"
    k_yaw: 0.01
    yaw_deadband_px: 60
    v_min: 0.08
    stop_area: 0.9
    max_lin_vel: 0.5
    max_ang_vel: 0.05
    search_ang_vel: 0.2
    max_lost: 50
    min_iou_keep: 0.01
    use_lidar_stop: false
    lidar_topic: "/scan"
    safe_dist: 0.35